tensorflow/core/kernels/training_ops.cc

```diff
@@ -310,21 +311,62 @@ struct ApplyAdamNonCuda {
                   typename TTypes<T>::ConstScalar beta2,
                   typename TTypes<T>::ConstScalar epsilon,
                   typename TTypes<T>::ConstFlat grad, bool use_nesterov) {
+    // Get params length and check if they can be vectorized by packet size.
+    Index length = var.size();
+    Index size = Eigen::internal::packet_traits<T>::size;
```

Comment:
`packet_size` is more common name for this variable in Eigen/TF codebase

Reply:
Done.